guides/09_other-tutorials/running-gradio-on-your-web-server-with-nginx.md

```diff
@@ -48,14 +48,20 @@ server {
 }
 ```
 
-
```

Comment:
bring back this empty line, otherwise there are annoying rendering issues with Tip. I should really figure that out. 

<img width="846" alt="Screen Shot 2024-07-01 at 10 46 32 AM" src="https://github.com/gradio-app/gradio/assets/9021060/2e30512d-8eb9-40c9-9135-acec0f0a6fa7">

vs 
<img width="869" alt="Screen Shot 2024-07-01 at 10 47 15 AM" src="https://github.com/gradio-app/gradio/assets/9021060/1fb00f85-987a-421a-87e9-d3655095f8a2">